lib/faraday/parameters.rb

```diff
@@ -63,6 +68,11 @@ def self.encode(params)
       return buffer.chop
     end
 
+    # @param query [nil, String]
+    #
+    # @raise [TypeError] if the nesting is incorrect
+    #
+    # @return [Array<Array, String>] the decoded params
```

Comment:
I was looking for a convention on tags order, but I couldn't find anything.
At least I think we should try to be consistent. I prefer the order of the method above so can we align this one to it (`@param`, `@return`, `@raise`)?

Reply:
👍